HelpSource/Classes/Nil.schelp

```diff
@@ -7,6 +7,16 @@ description::
 Nil has a single instance named nil and is used to represent uninitialized data,
 bad values, or terminal values such as end-of-stream.
 
+note::
+Nil is the object representation of code::nil::. Comparing it against code::nil:: will not produce the results you expect: Nil, the object, can not be nil. You are not expected to use this class directly: use code::nil::
```

Comment:
- Maybe you can write `Nil is the object representation of the value code::nil::.`? Is it clearer to say "the value nil"?
- There is one spot where `nil` is not written `code::nil::`

Reply:
Thanks Patrick - will make that and a few other changes too, I tried to do this too quickly!